helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -99,9 +99,15 @@ internal class CreateCommand : CliktCommand(
 
     private val nonOffendingLicenseCategories by option(
         "--non-offending-license-categories",
-        help = "Configure the path exclude generation to not create excludes for files or directories which only " +
-                "contain non-offending licenses specified by this comma separated list of license category names. " +
-                "Each category name must be present in the given license classifications file."
+        help = "Specify license IDs by their category which should be considered non-offending. Path excludes are not" +
```

Comment:
I believe "IDs" should be dropped here as that's not what's being specified, so it should read "Specify licenses by their category ..."